content/blog/2022-02-21-vorgehen-vermitteln/vorgehen-vermitteln.md

```diff
@@ -0,0 +1,49 @@
+---
+title: So lernen Kunden unser Vorgehen kennen und schätzen
+slug: so-lernen-kunden-unser-vorgehen-software-entwicklung-kennen-und-schaetzen
+templateKey: blog-post
+image: hero-vorgehen.jpg
+date: 2022-02-21
+author: Philip Schönholzer
+description: >-
+  Wir haben unser Vorgehen im Bereich Softwareentwicklung jahrelang verbessert und weiterentwickelt. Dabei blieben die Kunden häufig auf der Strecke. Für uns war sonnenklar, wie ein Projekt entwickelt werden sollte. Die Kunden aber waren unsicher, was sie erwartet und was wann passiert.
+categories:
+  - Allgemein
+---
+
+Unser Vorgehen für die Softwareentwicklung wurde im Jahre 2008 von der agilen Welle stark beeinflusst. Ebenso das Vorgehen "Lean Startup". Immer mehr Techniken und Tools setzten wir ein, um die Projekte optimal zum Abschluss zu bringen. Unsere Kunden verstanden aber häufig bloss Bahnhof. Wir haben vergessen, dass unsere Kunden nicht dieselbe Lernkurve wie wir genommen haben, und liessen sie entsprechend im Dunkeln tappen. Dies war für sie natürlich unangenehm. Wenn wir ihnen Resultate an einem Review-Meeting zeigten, war ihnen unklar, was sie sahen und auch was als nächstes passiert. Entsprechend haben wir von Kunden immer mal wieder gehört, dass unser Vorgehen undurchsichtig ist. Wir mussten etwas unternehmen.
```

Comment:
Ich habe mich gefragt: Was genau ist im Jahr 2008 passiert? Wie wäre folgende Formulierung für die ersten beiden Sätze?
_Unser Vorgehen für die Softwareentwicklung wurde in den vergangenen Jahren von der agilen Welle und der Methode "Lean Startup" stark beeinflusst._
Für die Agile Welle und Lean Startup würden sich Fussnoten auch noch eignen:
https://de.wikipedia.org/wiki/Agile_Softwareentwicklung
https://de.wikipedia.org/wiki/Lean_Startup